.github/CONTRIBUTING.md

```diff
@@ -17,6 +17,9 @@ Thank you for showing interest in contributing to Pretendo Network! As an open s
 - [Licensing](#licensing)
 
 # Prerequisites
+> [!CAUTION]
+> You may only contribute to Pretendo Network if you are legally able to do so. Being a part of the Nintendo Developer program, being under any other related NDA, having access to any related SDKs legally obtained or not, etc. prohibits you from contributing to Pretendo Network. If it is discovered that you are a part of these programs, are under any related NDA, have access to any related SDKs legally obtained or not, etc. you will immediately be blocked from the organization, even if you never use any information in contributions or don't even use the mentioned tools.
+
```

Comment:
```suggestion
> [!CAUTION]
> By contributing to this project, you agree to the following:  
>
> 1. **Developer Certificate of Origin** – All contributions must comply with the [Developer Certificate of Origin](https://developercertificate.org/).  
> 2. **Ownership & Rights** – You may only contribute content you own. Do not submit anything that is:  
>    - Owned by someone else (unless properly licensed for this project).  
>    - A trade secret, under NDA, or otherwise confidential.  
> 3. **Respect for Nintendo’s IP** – We strictly avoid infringing on Nintendo’s intellectual property. If you have had access to Nintendo’s proprietary materials (legally or otherwise), we may take steps to ensure compliance, including blocking contributions or removing existing ones.  
```